### PR TITLE
OpenMS: Downgrade and fix Xerces to 3.1.4

### PR DESCRIPTION
**Pull request by Leon-Bichmann**
Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it. Note that if you are not already a member of the bioconda project (meaning that you can't add this label), please ping `@bioconda/core` so that your PR can be reviewed and merged (please note if you'd like to be added to the bioconda project). Please see https://github.com/bioconda/bioconda-recipes/issues/15332 for more details.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [x] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/openms/meta.yaml [hunk 1]
```diff
@@ -22,9 +22,9 @@ requirements:
     - python
     - zlib
     - libtool
-    - xerces-c
+    - xerces-c=3.1
```
Comment (oliveralka) [inline recipes/openms/meta.yaml]: I guess you have to specify via bool '=='

Comment (Leon-Bichmann) [reply recipes/openms/meta.yaml]: thx

**Comment by oliveralka** [comment]
```
Traceback (most recent call last):
  File "/home/circleci/project/miniconda/lib/python3.7/site-packages/conda/models/version.py", line 38, in __call__
    return cls._cache_[arg]
KeyError: '>=5.5'
```
```
 File "/home/circleci/project/miniconda/lib/python3.7/site-packages/conda/models/version.py", line 171, in __init__
    raise InvalidVersionSpec(vstr, "invalid character(s)")
conda.exceptions.InvalidVersionSpec: Invalid version '>=5.5': invalid character(s)
```

File: recipes/openms/meta.yaml (hunk 1 above)
Comment (oliveralka) [reply recipes/openms/meta.yaml]: ok, this was alright - sry!

**Pull request closed without merging**